hasher-matcher-actioner/webapp/src/pages/settings/ActionRuleSettingsTab.tsx

```diff
@@ -136,66 +132,32 @@ export class ActionRule {
       );
 }
 
-export type Action = {
-  name: string;
-  id: string;
-};
-
 const defaultActionRule = new ActionRule(
   '',
   [{key: classificationTypeTBD, value: ''}],
   [],
-  '0',
+  {key: 'ActionLabel', value: '0'},
```

Comment:
`'0'` feel pretty dubious (not blocking as it already exists but having this work with null or undefined would be preferred)

Reply:
Looking at more refactoring of action rules later and will incorporate this then